docs/en/writing-running-appium/ios/ios-xctest-mobile-gestures.md

```diff
@@ -243,24 +243,29 @@ params.put("element", ((RemoteWebElement) element).getId());
 js.executeScript("mobile: selectPickerWheelValue", params);
 ```
 
+### mobile: rotateElement
 
-### mobile: alert
```

Comment:
Will https://github.com/appium/appium-xcuitest-driver/blob/8632cc98e2eee7c5875d79f92ecf38cfdd42acf5/lib/commands/alert.js#L31 be removed?

Reply:
ah, i see. This doc is for gestures.